benchmarks/bench_storage.c

```diff
@@ -22,15 +23,17 @@ static __thread unsigned int rseed = 1234; /* XXX: make this an option */
 } while (0)
 
 #define BENCHMARK_OPTION(ACTION)\
-    ACTION(entry_min_size,  OPTION_TYPE_UINT, 64,    "Min size of cache entry")\
-    ACTION(entry_max_size,  OPTION_TYPE_UINT, 64,    "Max size of cache entry")\
-    ACTION(nentries,        OPTION_TYPE_UINT, 1000,  "Max total number of cache entries" )\
-    ACTION(nops,            OPTION_TYPE_UINT, 100000,"Total number of operations")\
-    ACTION(pct_get,         OPTION_TYPE_UINT, 80,    "% of gets")\
-    ACTION(pct_put,         OPTION_TYPE_UINT, 10,    "% of puts")\
-    ACTION(pct_rem,         OPTION_TYPE_UINT, 10,    "% of removes")
-
-#define O(b, opt) option_uint(&(b->options.benchmark.opt))
+    ACTION(entry_min_size,    OPTION_TYPE_UINT, 64,    "Min size of cache entry")\
+    ACTION(entry_max_size,    OPTION_TYPE_UINT, 64,    "Max size of cache entry")\
+    ACTION(nentries,          OPTION_TYPE_UINT, 1000,  "Max total number of cache entries" )\
+    ACTION(nops,              OPTION_TYPE_UINT, 100000,"Total number of operations")\
+    ACTION(pct_get,           OPTION_TYPE_UINT, 80,    "% of gets")\
+    ACTION(pct_put,           OPTION_TYPE_UINT, 10,    "% of puts")\
+    ACTION(pct_rem,           OPTION_TYPE_UINT, 10,    "% of removes")\
+    ACTION(storage_cfg_path,  OPTION_TYPE_STR , NULL,  "Path to storage config")
```

Comment:
In the original version of this benchmark, the config file for various modules was composable, i.e., you could have made a config with options for the benchmark alongside options for the storage engine.
Is that no longer true?

Reply:
I introduce regression in cd767666539f2aca9b461bbfb28ac3cc47b1be39 - sorry about that. I will fix benchmarks in separate PR.